js/cards.js

```diff
@@ -86,8 +83,13 @@ $(document).ready(() => {
   };
 
   function renderCard(data) {
-    let card = `<div id="${data.id}${data.card}" class="card-common card-${data.card}">${data.name}</div>`;
-    $("#cards").append(card);
+    if (data.type === 'raise') {
+      let card = `<div id="${data.id}${data.card}" class="card-common card-${data.card}">${data.name}</div>`;
+      $("#cards").append(card);
+    } else if (data.type === 'lower') {
+      $(`#${data.id}${data.card}`).remove();
```

Comment:
using `data.id` as selector breaks when the `id` contains certain characters (`.`, `@`)